tests/extract/test_extract_job.sql

```diff
@@ -47,16 +47,6 @@ FROM
             `test_treeherder`.`bug_job_map` AS `t2` ON `t2`.`job_id` = `t1`.`id`
         UNION ALL
 
-        SELECT
```

Comment:
@klahnakoski the `test_extract_job_sql` is failing but it's unclear to me why. Since you wrote this test and the associated code could you give me some insight?

Reply:
These tests were designed to fail when the schema changes.  The solution is to review the changes and then replace the old SQL with new SQL.

Reply:
https://github.com/mozilla/treeherder/pull/6432/files

Reply:
The reason for these tests is to ensure the extract to BigQuery stays synchronized with the new schema. In theory the code can handle schema changes, but this ensures a human gets to review them.  Once you have updated the `*.sql` and `*.json` files, your PR will include the extracted changes for review.

Reply:
That worked, thanks.